docs/_static/niscope_usage.inc

```diff
@@ -13,7 +13,7 @@ The following is a basic example of using the **niscope** module to open a sessi
         with session.initiate():
             waveforms = session.channels[0,1].fetch(num_records=5)
         for wfm in waveforms:
-            print('Channel {0}, record {1} samples acquired: {2:,}\n'.format(wfm.channel, wfm.record, len(wfm.wfm)))
+            print('Channel {0}, record {1} samples acquired: {2:,}\n'.format(wfm.channel, wfm.record, len(wfm.waveform)))
```

Comment:
based on this... 
Do you think the member should be renamed `waveform` -> `data` or something like that?

Reply:
Ok, so we have `WaveformInfo` and `samples`. We don't have anything called `Waveform`.
Question:

* Should `WaveformInfo` be called `Waveform`? It would be further deviation from C API.

Introducing a `Waveform` type is a big commitment although this would be actually scoped to `niscope.Waveform`.

What do you think?

Reply:
I think we should not use Waveform. It is too generic for this. What we return is very specific to niscope and if we ever were to create a common waveform data type, then we wouldn't be able to use the name Waveform without a lot of confusion. "Which Waveform type are talking about?"

Reply:
ok